coderadar-rest/src/main/java/io/reflectoring/coderadar/rest/query/GetCriticalFilesController.java

```diff
@@ -0,0 +1,32 @@
+package io.reflectoring.coderadar.rest.query;
+
+import io.reflectoring.coderadar.query.domain.ContributorsForFile;
+import io.reflectoring.coderadar.query.port.driver.GetCriticalFilesUseCase;
+import java.util.List;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@Transactional
+public class GetCriticalFilesController {
+  private final GetCriticalFilesUseCase getCriticalFilesUseCase;
+
+  public GetCriticalFilesController(GetCriticalFilesUseCase getCriticalFilesUseCase) {
+    this.getCriticalFilesUseCase = getCriticalFilesUseCase;
+  }
+
+  @GetMapping(
+      path = "/projects/{projectId}/files/critical",
+      produces = MediaType.APPLICATION_JSON_VALUE)
+  public ResponseEntity<List<ContributorsForFile>> getCriticalFiles(
```

Comment:
The parameter numOfContr can simply be an `int`

Reply:
Done.